src/mmw/apps/modeling/mapshed/tasks.py

```diff
@@ -138,6 +140,20 @@ def collect_data(geop_result, geojson):
     z.CN = np.array(geop_result['cn'])
     z.SedPhos = geop_result['sed_phos']
 
+    # Additional calculated values
+    basin_area_sq_km = (area / 1000000)  # Square Kilometers
```

Comment:
Consider adding `SQKM_PER_SQM` as a constant, like this: https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/apps/modeling/mapshed/tasks.py#L37, and multiplying with `area` instead of dividing.

```python
area_sq_km = area * SQKM_PER_SQM
```